app/presenters/speedy_af/proxy/derivative.rb

```diff
@@ -18,4 +18,8 @@ class SpeedyAF::Proxy::Derivative < SpeedyAF::Base
   def bitrate
     audio_bitrate.to_i + video_bitrate.to_i
   end
+
+  def download_path
+    Avalon::Configuration.construct_download_path.call(self)
+  end
```

Comment:
Since this is the same as in the Derivative model, how about moving it out to `DerivativeBehavior`?